controllers/datamovement_controller.go

```diff
@@ -51,6 +56,19 @@ const (
 	finalizer = "dm.cray.hpe.com"
 
 	InitiatorLabel = "dm.cray.hpe.com/initiator"
+
+	// DM ConfigMap Info
+	DMConfigMapName      = "nnf-dm-config"
```

Comment:
The "DM" here seems redundant. It's already in the `data movement` package

Reply:
I don't think these need to be exported either

Reply:
Thanks. And you're right - I thought I needed to export them for the tests, but that's not the case.